source/Calamari.Tests/Fixtures/StructuredVariables/Approved/YamlVariableReplacerFixture.ShouldExpandOctopusVariableReferences.approved.yaml

```diff
@@ -0,0 +1,20 @@
+﻿server:
+  ports:
+  - "8080"

```

Comment:
Should this be a string, or int?

Reply:
The input is `"5555"`, so the replacer preserves the type at that position. I deliberately include a few ambiguous-looking inputs, to help confirm the implementation doesn't get thrown by them.

Reply:
I suppose it does come at the cost of not reading like a "normal" config file.